Dockerfile-linux.template

```diff
@@ -166,6 +163,8 @@ RUN set -eux; \
 			apt-get install -y --no-install-recommends -t "$VERSION_CODENAME-backports" golang-go; \
 		); \
 {{ ) end -}}
+# TODO consider CGO_ENABLED=0 instead of installing gcc (probably doesn't matter much as this whole block is 1.20-only and I *think* newer versions already *do* default to cgo-disabled for builds, but maybe that's just for the dist/release builds)
```

Comment:
aha, yes: https://github.com/golang/go/blob/b44f6378233ada888f0dc79e0ac56def4673d9ed/src/cmd/dist/build.go#L1312

however, this was only added in https://github.com/golang/go/commit/da0c375c571037ec5ea6f8ef2be8f07593b40eb6, so only since Go 1.21
(so this is only something to keep in mind when we revisit #464)

Reply:
Adjusted the comment to be better